lib/plugins/babelRuntimePathResolver.js

```diff
@@ -2,11 +2,12 @@
 // in OBT's node_modules directory and not the module's
 'use strict';
 
-const path = require('path');
 const falafel = require('falafel');
 
 const prefix = /babel-runtime/;
-const rootDirectory = path.resolve(__dirname, '../../node_modules');
+
+// HACK: Can't require.resolve babel-runtime so use babel-core instead
+const rootDirectory = require.resolve('babel-core').replace(/(.*node_modules).*/, '$1');
```

Comment:
I'm not sure. What happens if your `rootDirectory` resolves to be the `node_modules` within obt (because user is on npm v2, or just npm v3 with obt linked to a global, say), but then I want to require something that is not within obt but within my module's own `node_modules` dir? Won't it fail in that case?